ci/infra/testrunner/tests/test_dockercaps.py

```diff
@@ -0,0 +1,77 @@
+import logging
+import os
+import pytest
+import pathlib
+import textwrap
+
+logger = logging.getLogger("testrunner")
+
+
+def test_dockercaps(deployment, kubectl):
+
+    CONTENT = """---
```

Comment:
If you extract the constant to the module-level, it won't break the visual indentation of the function.

This would help the code to be more readable.